ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -21,12 +21,12 @@ open class ContactView: UIControl {
         case large
         case small
 
-        var avatarSize: AvatarLegacySize {
+        var avatarSize: MSFAvatarSize {
             switch self {
             case .large:
-                return .extraExtraLarge
+                return .xxlarge
```

Comment:
Nit: capitalize the L in large to reflect Hunter's changes (#514)?

Reply:
Thanks! I'll do that in a separate PR to normalize all of these.